pyspecProcScripts/first_level/QESR_rescale.py

```diff
@@ -1,43 +1,106 @@
 from pyspecdata.datadir import pyspec_config # piggyback on _pyspecdata
-from pyspecdata.general_functions import strm
```

Comment:
you need to merge master into this branch.  I ran black on master, but you  need to resolve the conflicts.